Fails to compile

```
go get github.com/iawia002/annie   
# github.com/iawia002/annie/extractors/youtube
go/src/github.com/iawia002/annie/extractors/youtube/youtube.go:122:37: not enough arguments in call to ytdl.GetVideoInfo
        have (string)
        want (context.Context, interface {})
go/src/github.com/iawia002/annie/extractors/youtube/youtube.go:190:27: videoInfo.GetDownloadURL undefined (type *ytdl.VideoInfo has no field or method GetDownloadURL)
```

I'm having the same problem (go version 1.14) --

```
$ go get github.com/iawia002/annie
# github.com/iawia002/annie/extractors/youtube
/home/user/gocode/src/github.com/iawia002/annie/extractors/youtube/youtube.go:122:37: not enough arguments in call to ytdl.GetVideoInfo
        have (string)
        want (context.Context, interface {})
```

--- a/README.md
+++ b/README.md
@@ -64,7 +64,7 @@ The following dependencies are required and must be installed separately.
 To install Annie, use `go get`, or download the binary file from [Releases](https://github.com/iawia002/annie/releases) page.
 
 ```bash
-$ go get github.com/iawia002/annie
+$ GO111MODULE=on go get github.com/iawia002/annie
 ```
 
 ### Homebrew (macOS only)

